auction_keeper/urn_history.py

```diff
@@ -57,17 +57,25 @@ def get_urns(self) -> Dict[Address, Urn]:
         # Get a unique list of urn addresses
         from_block = max(0, self.cache_block - self.cache_lookback)
         to_block = self.web3.eth.blockNumber
-        frobs = self.mcd.vat.past_frobs(from_block=from_block, to_block=to_block, ilk=self.ilk,
-                                        chunk_size=self.chunk_size)
-        for frob in frobs:
-            urn_addresses.add(frob.urn)
+        logs = self.mcd.vat.past_logs(from_block=from_block, to_block=to_block, ilk=self.ilk,
+                                      chunk_size=self.chunk_size)
+        for log in logs:
+            if isinstance(log, Vat.LogFrob):
+                urn_addresses.add(log.urn)
+            if isinstance(log, Vat.LogFork) or isinstance(log, Vat.LogMove):
+                urn_addresses.add(log.src)
```

Comment:
A `src` that is forking some of its debt to `dst` would have needed to previously `frobbed`, so recording `log.src` may not be needed, as `src` would have been captured in a `LogFrob`.

Reply:
I considered the case the urn A was forked to urn B which was forked to urn C.  In that case, urn B would never have been frobbed, but it also would've been the `dst` of the first fork, so it'd be captured anyway.  As such; I agree.  I'll remove line 66 and retest.

Reply:
Retested; no issues with the change.